sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -162,6 +172,46 @@ private void initializeSkin(Application app, OoyalaSkinLayout l, OoyalaPlayer p,
     l.addView(rootView, frameLP);
   }
 
+  private void overrideLocale(JSONObject configJson, Context context) {
+    String locale = Locale.getDefault().getLanguage();
+    try {
+      configJson.put("locale", locale);
```

Comment:
It looks like you are replacing the "locale" in skin.json, to be the locale in the device.  I would say this is not what we want to do by default.